chains/evm/contracts/pools/USDC/SiloedUSDCTokenPool.sol

```diff
@@ -89,35 +90,26 @@ contract SiloedUSDCTokenPool is SiloedLockReleaseTokenPool, AuthorizedCallers {
 
     _validateReleaseOrMint(releaseOrMintIn, localAmount, WAIT_FOR_FINALITY);
 
-    // Save gas by using storage instead of memory as a value may need to be updated.
-    SiloConfig storage remoteConfig = s_chainConfigs[releaseOrMintIn.remoteChainSelector];
-
     uint256 excludedTokens = s_tokensExcludedFromBurn[releaseOrMintIn.remoteChainSelector];
-
+    ERC20LockBox lockbox = _getLockBox(releaseOrMintIn.remoteChainSelector);
+    // No excluded tokens is the common path, as it means no migration has occured yet, and any released
+    // tokens should come from the stored token balance of previously deposited tokens.
     if (excludedTokens == 0) {
-      // No excluded tokens is the common path, as it means no migration has occured yet.
-      // Any released tokens should come from the stored token balance of previously deposited tokens.
-      if (localAmount > remoteConfig.tokenBalance) {
-        revert InsufficientLiquidity(remoteConfig.tokenBalance, localAmount);
+      uint256 availableLiquidity = i_token.balanceOf(address(lockbox));
```

Comment:
Feels like this check should always happen, not just here, as we withdraw from the lockbox regardless of excludedTokens. 

Also I'm not completely following how excludedTokens is doing anything here. Do we still need it?